lib/libutee/arch/riscv/utee_syscalls_rv64.S

```diff
@@ -0,0 +1,32 @@
+/* SPDX-License-Identifier: BSD-2-Clause */
+/*
+ * Copyright  2022  Beijing ESWIN Computing Technology Co., Ltd.
+ */
+
+#include <tee_syscall_numbers.h>
+#include <asm.S>
+
+        .section .text
+
+        .macro UTEE_SYSCALL name, scn, num_args
+	FUNC \name , :
+
+	.if \num_args > TEE_SVC_MAX_ARGS || \num_args > 8
+	.error "Too many arguments for syscall"
+	.endif
+	addi	sp, sp, -16
```

Comment:
Why are you doing anything with the stack pointer here?

Reply:
When there are more parameters, use the stack to pass two parameters。

Reply:
There will never be more than eight, I believe it should be possible to copy the pattern from `utee_syscalls_a64.S`.

Reply:
Following our original logic, RISC-V uses the a0-a7 register to pass parameters, 8 parameters, plus the system call number and number of parameters, at most 10 parameters need to be passed, so the stack is used to pass 2 parameters.  As you suggested, the number of arguments is not passed, and the system call number is passed as t0 so that stack memory is not used